charts/mlp/README.md

```diff
@@ -0,0 +1,72 @@
+# mlp
+
+![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![AppVersion: 1.7.1](https://img.shields.io/badge/AppVersion-1.7.1-informational?style=flat-square)
+
+MLP API
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| caraml-dev | <caraml-dev@caraml.dev> |  |
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.helm.sh/stable | postgresql | 7.0.0 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| externalPostgresql.address | string | `"127.0.0.1"` |  |
+| externalPostgresql.database | string | `"mlp"` |  |
+| externalPostgresql.password | string | `"password"` |  |
+| externalPostgresql.username | string | `"mlp"` |  |
+| mlp.apiHost | string | `"http://mlp/v1"` |  |
```

Comment:
let's remove `mlp` namespace for these values. the reason to that is because when this chart is included as a dependency there will be doubled namespace:
```yaml
mlp_dependency_name:
    mlp:
        apiHost: ...
        ...
```

Reply:
modified. congregated them under `deployment|service|ingress` based on where they are used, so that the parent chart would have

```
mlp_dependency_name:
    deployment:
        apiHost:...
        ...
```